integration/debug_test.go

```diff
@@ -55,23 +53,24 @@ func TestDebug(t *testing.T) {
 	}
 	for _, test := range tests {
 		t.Run(test.description, func(t *testing.T) {
+			// Run skaffold build first to fail quickly on a build failure
+			skaffold.Build(test.args...).InDir(test.dir).RunOrFail(t)
+
 			ns, client, deleteNs := SetupNamespace(t)
 			defer deleteNs()
 
-			stop := skaffold.Debug(test.args...).WithConfig(test.filename).InDir(test.dir).InNs(ns.Name).WithEnv(test.env).RunBackground(t)
+			stop := skaffold.Debug(test.args...).InDir(test.dir).InNs(ns.Name).RunBackground(t)
 			defer stop()
 
 			client.WaitForPodsReady(test.pods...)
-			client.WaitForDeploymentsToStabilize(test.deployments...)
```

Comment:
The `jib` example is a deployment.

Reply:
Not sure I understand your point

Reply:
By deleting this line, the deployments may not be stabilized before the test continues. Right?

Reply:
`GetDeployment` does it